_posts/2019-10-27-Merging-Multiple-SimpleCov-coverage-results.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Merging-Multiple-SimpleCov-coverage-results
+date: 2019-10-29 13:56:00
+categories: ["Rails", "Simplecov", "Upgrades"]
+author: bronzdoc
+---
+
+As part of our Roadmap service at FastRuby.io, we have to analyze the test suite of the application we are upgrading to give a proper estimate on how long it will take us to upgrade. We use SimpleCov for this.
+We got in to the problem of having SimpleCov results across different containers for the same application. Our goal was to have a single result for the whole application, so in this blog post we are going to show you how we solved that problem.
+
+<!--more-->
+
+Some of the applications we upgrade are outdated and setups can be difficult, with no documentation, missing steps on how to set it up, etc. Even after the setup, the tests of these applications can take several hours to complete and generate a coverage report.
```

Comment:
Please try to avoid using `etc.` in your articles. You can rephrase it like this:

```
Some of the applications we upgrade are outdated and setting them up can be difficult. Sometimes they have no documentation, missing steps on how to set them up, or outdated installation steps.
```

Reply:
This could be improved: 

```
Even after the setup, the tests of these applications can take several hours to complete and generate a coverage report.
```

Like this:

```
Even after setup, the test suite of these applications can take several hours to complete and generate a coverage report.
```